### ISSUE TITLE
WARNING: non-finite loss, ending training 

### ISSUE THREAD

**Issue by CheungBH** [coldlarry/YOLOv3-complete-pruning#92]
Hello. When I am doing quantizing with the dataset "oxford hand", 

_python3 train.py --data data/oxfordhand.data --batch-size 32 --accumulate 1 --weights weights/yolov3.weights --cfg cfg/yolov3-quantize-hand.cfg_

the prediction of the model is always **-inf** and the loss comes to **nan**.
I wonder what's happening and how to modify it? I used yolov3.weights and weights/darknet53.conv.74 as the pre-trained models and the results are always the same. 

![image](https://user-images.githubusercontent.com/39186017/93010179-13056980-f5bc-11ea-836a-540498b1e0b6.png)

**Comment by CheungBH** [comment]
I fount that when the w_bit and a_bit is set to be 16, the output will be -inf in **quantized conv** when running the **forward()**:

    _def forward(ctx, input, nbit):
        scale = 2 ** nbit - 1
        return torch.round(input * scale) / scale_

And when the bits are set to be 8, the code can be run as usual. 

Actually, when the a-bit is 16, the scale is 65535.
